hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieDataFile.java

```diff
@@ -18,53 +18,53 @@
 
 import com.uber.hoodie.common.util.FSUtils;
 import java.io.Serializable;
-import java.util.Comparator;
 import org.apache.hadoop.fs.FileStatus;
 
 public class HoodieDataFile implements Serializable {
```

Comment:
Made HoodieFileGroup and its containing objects (HoodieDataFile/HoodieLogFile) serializable